picard/coverart/image.py

```diff
@@ -282,7 +282,7 @@ def save(self, dirname, metadata, counters):
         if not self.can_be_saved_to_disk:
             return
         if (config.setting["caa_image_type_as_filename"] and
-            not self.is_front_image()):
+                not self.is_front_image()):
```

Comment:
In this case, i prefer to have the `and` at start of the line:

```python
if (config.setting["caa_image_type_as_filename"]
    and not self.is_front_image()):
```

Reason: having all logical operators aligned vertically is much easier to read.

```python
if (expr1
    or not expr2
    and (expr3
         or expr4)
    and expr4):
```

Also it is consistent with http://legacy.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator